src/axom/core/utilities/StringUtilities.cpp

```diff
@@ -39,6 +39,16 @@ void toUpper(std::string& str)
   });
 }
 
+bool checkedConvertToInt(const std::string& number, int& result)
+{
+  // Use the C versions to avoid the exceptions
+  // thrown by std::stoi on conversion failure
+  // FIXME: Switch to std::from_chars when C++17 is available
+  char* ptr;
+  result = strtol(number.c_str(), &ptr, 10);
+  return *ptr == 0;
+}
```

Comment:
fyi: Conduit already provides both a way to check if a string is a number and a method that does the conversion in its `utils`. 

https://github.com/LLNL/conduit/blob/f9b61cbadd270dd1ef8c7987e63e3f509951c768/src/libs/conduit/conduit_utils.cpp#L956

https://github.com/LLNL/conduit/blob/f9b61cbadd270dd1ef8c7987e63e3f509951c768/src/libs/conduit/conduit_utils.hpp#L443

Reply:
Thanks for the suggestion - always good to get rid of duplicated functionality!